pkg/middleware/auth_proxy.go

```diff
@@ -55,16 +55,16 @@ func initContextWithAuthProxy(store *remotecache.RemoteCache, ctx *models.ReqCon
 		return true
 	}
 
-	id, err := logUserIn(auth, username, logger, false)
-	if err != nil {
-		ctx.Handle(407, err.Error(), err.DetailsError)
+	id, e := logUserIn(auth, username, logger, false)
```

Comment:
err is renamed to e since it's not of the error type, which is expected of variables named err.